.github/workflows/docs_site.yml

```diff
@@ -0,0 +1,62 @@
+# Sample workflow for building and deploying a Jekyll site to GitHub Pages
+name: Deploy Jekyll with GitHub Pages dependencies preinstalled
+
+on:
+  # Runs on pushes targeting the default branch
+  push:
+    branches: ["main"]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+# Allow only one concurrent deployment, skipping runs queued between the run in-progress and latest queued.
+# However, do NOT cancel in-progress runs as we want to allow these production deployments to complete.
+concurrency:
+  group: "pages"
+  cancel-in-progress: false
+
+jobs:
+  # Build job
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.10'
+          cache: 'pip'
+      - name: Build Mkdocs
+        run: |
+            pip install mkdocs mkdocs-awesome-nav
+            mkdocs build
+      - uses: actions/setup-java@v4
+        with:
+          java-version: '21'
+          distribution: 'adopt'
+          cache: 'maven'
+      - name: Build Javadoc and Enunciate Swagger UI
+        run: |
+            mvn --batch-mode -Preproducible compile javadoc:javadoc javadoc:aggregate
+            cp -r target/reports/apidocs site/apidocs
+            cp -r proxyserver/target/enunciate-docs/apidocs site/endpoints
+            # Add a setting to swagger-ui to turn off the "Try it now" buttons
```

Comment:
Aw dang. I can understand why, though. Is there a way to see the curl request even without the "Try it now" button?

Reply:
haha, I saw how much you liked the curl commands and knew you would be disappointed here. I disabled it because there's no chance that trying it out in the browser will work (CORS issues, and I don't want to loosen CORS restrictions on the server), but I do agree the curl commands are useful.

If you don't think the failures are confusing, I can keep the button around. I'll remove the PROD server URL though since I don't want to directly enable folks to call it through the browser.